testsuite/forge-cli/src/main.rs

```diff
@@ -1899,7 +1899,7 @@ fn pfn_const_tps(
         .with_initial_validator_count(NonZeroUsize::new(7).unwrap())
         .with_initial_fullnode_count(7)
         .with_emit_job(EmitJobRequest::default().mode(EmitJobMode::ConstTps { tps: 100 }))
-        .add_network_test(PFNPerformance::new(add_cpu_chaos, add_network_emulation))
+        .add_network_test(PFNPerformance::new(2, add_cpu_chaos, add_network_emulation))
```

Comment:
Why are we reducing this? Do you require it for more accurate mempool experiments? If so, could we make this configurable? It's helpful on the state sync side because it allows us to test more cases.